server/src/pages/shot/view.js

```diff
@@ -366,6 +366,14 @@ class Body extends React.Component {
     let downloadButton = null;
     let copyButton = null;
 
+    downloadButton = <div className="download-shot-button">
+      <Localized id="shotPageDownloadShot" attrs={{title: true}}>
+        <button className={`button transparent nav-button`} onClick={this.onClickDownload.bind(this)}
+                title="Download the shot image">
+          <img src={this.props.staticLink("/static/img/icon-download.svg")} />
+        </button>
+      </Localized></div>;
+
     if (this.props.isOwner) {
```

Comment:
This check isn't elegant but keeping it in favor of executing a block of code that's not needed for !owned shots